solidity/contracts/TokenStaking.sol

```diff
@@ -60,8 +60,8 @@ contract TokenStaking is Authorizations, StakeDelegatable {
     event ExpiredLockReleased(address indexed operator, address lockCreator);
 
     uint256 public minimumStakeScheduleStart;
-    uint256 public initializationPeriod;
-    uint256 public undelegationPeriod;
+    uint256 public initializationPeriod; // varies between mainnet and testnet
+    uint256 public constant undelegationPeriod = 5184000; // ~60 days
```

Comment:
I'm not 100% happy with this change from testing perspective, but I can live with it.

Reply:
Well... it does not change anything that much and there is also a bright side: we are testing against the value close to the real one.

Reply:
It may complicate future system tests a little bit when we won't be able to tweak the value for specific environment or test case.

Reply:
Agreed but I have a nice surprise for you: https://github.com/keep-network/keep-core/issues/1900

Not sure how it's going to look like yet but I plan to work on it this week.

Reply:
Props!